DesktopEdge/Views/Screens/IdentityDetails.xaml.cs

```diff
@@ -46,7 +46,7 @@ public partial class IdentityDetails : UserControl {
         public delegate void MFAToggled(bool isOn);
         public event MFAToggled OnMFAToggled;
         public delegate void Detched(MouseButtonEventArgs e);
-        public event Detched OnDetach;
+        public event Detched HandleAttachment;

```

Comment:
Name changed like on the main screen for this functionality of attach/detachment for the window.